packages/trackerless-network/src/logic/protocol-integration/stream-message/StreamMessageTranslator.ts

```diff
@@ -67,25 +93,25 @@ export class StreamMessageTranslator {
                 sequenceNumber: msg.getPreviousMessageRef()!.sequenceNumber,
                 streamId: msg.getStreamId() as string,
                 streamPartition: msg.getStreamPartition(),
-                publisherId: msg.getPublisherId(),
+                publisherId: toBinary(msg.getPublisherId()),
                 messageChainId: msg.getMsgChainId()
             }
         }
         let newGroupKey: EncryptedGroupKey | undefined = undefined
         if (msg.getNewGroupKey()) {
             newGroupKey = {
-                encryptedGroupKeyHex: msg.getNewGroupKey()!.encryptedGroupKeyHex,
-                groupKeyId: msg.getNewGroupKey()!.groupKeyId,
-                serialized: msg.getNewGroupKey()!.serialized ?? undefined
+                data: toBinary(msg.getNewGroupKey()!.encryptedGroupKeyHex),
+                groupKeyId: msg.getNewGroupKey()!.groupKeyId
             }
         }
         const translated: StreamMessage = {
```

Comment:
It is easiet to read the code (and not forget any field) if the fields are in the same order as in StreamMessage class itself.